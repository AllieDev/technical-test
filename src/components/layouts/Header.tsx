import * as React from "react";
import AppBar from "@mui/material/AppBar";
import Box from "@mui/material/Box";
import Toolbar from "@mui/material/Toolbar";
import Typography from "@mui/material/Typography";
import Button from "@mui/material/Button";
import IconButton from "@mui/material/IconButton";
import MenuIcon from "@mui/icons-material/Menu";

export default function ButtonAppBar() {
  const navItems = ["Home", "About", "Contact"];

  return (
    <Box sx={{ flexGrow: 1 }}>
      <AppBar position="static">
        <Toolbar>
          <Typography variant="h6" component="div" sx={{ flexGrow: 1 }}>
            Arcgis Map
          </Typography>
          <Box sx={{ display: { xs: "none", sm: "block" } }}>
            {navItems.map((item) => (
              <Button key={item} sx={{ color: "#fff" }}>
                {item}
              </Button>
            ))}
          </Box>
        </Toolbar>
      </AppBar>
    </Box>
  );
}
