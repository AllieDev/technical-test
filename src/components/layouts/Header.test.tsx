import Header from "./Header";
import { describe, expect, it } from "vitest";
import { render } from "@testing-library/react";

describe("Header", () => {
  it("should render", () => {
    const { container } = render(<Header />);
    expect(container.textContent).toMatchSnapshot();
  });
});
