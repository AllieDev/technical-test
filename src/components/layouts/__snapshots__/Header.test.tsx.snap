// Vitest Snapshot v1

exports[`Header > should render 1`] = `"Arcgis MapHomeAboutContact"`;
